DHL/JSON/9df77954-633e-4a19-8e78-a2e73e34050f.json

```diff
@@ -0,0 +1,998 @@
+[
+  {
+    "ShpInfReqDef": {
+      "Id": " 9df77954-633e-4a19-8e78-a2e73e34050f",
+      "Nm": "Returned Export - GR",
+      "Dsc": "This import shipment is a returning commercial export",
+      "DrvdFromShpInfReqDef": {},
+      "LedOwnOrgn": {
+        "CtryCd": "GR"
+      },
+      "ApplyAllInd": false,
+      "PrsnDatSenstyClssLvl": "Low",
+      "QuickGuideEnabledInd": false,
+      "UICmpt": {
+        "CmptNm": "RetExport",
+        "DisplayTtl": "Returned Export",
+        "Tooltip": "",
+        "CmptTy": "Composed",
+        "DisplayDsc": {
+          "DisplayTxt": ""
+        },
+        "Hint": ""
+      },
+      "Ent": [
+        {
+          "Id": "8453c6a8-6fa0-494e-9a19-fcb606f2b4ee",
+          "Nm": "Returned Export entity",
+          "Dsc": "Returned Export Discriptiption ",
+          "isImg": false,
+          "isExcl": false,
+          "isMan": true,
+          "Ent": [
```

Comment:
we need to add att "Customer role type code" refer 66da file.

Reply:
Done